tuhi/gui/drawing.py

```diff
@@ -18,13 +18,15 @@
 from pathlib import Path
 from .config import Config
 from tuhi.svg import JsonSvg
+from tuhi.png import JsonPng
 
 import gi
 gi.require_version("Gtk", "3.0")
 from gi.repository import GObject, Gtk, GdkPixbuf, Gdk  # NOQA
 
 
 DATA_PATH = Path(xdg.BaseDirectory.xdg_cache_home, 'tuhi', 'svg')
+PNG_DATA_PATH = Path(xdg.BaseDirectory.xdg_cache_home, 'tuhi', 'png')
```

Comment:
rename `DATA_PATH` to `SVG_DATA_PATH` for consistency please (could be done in a separate commit before this one).